### DISCUSSION TITLE
Solidus v4.1

### DISCUSSION THREAD

**Discussion by kennyadsl** [solidusio/solidus#5195] [Announcements]
Hello! 👋

We are delighted to inform you that the Solidus team has just released Solidus v4.1. In accordance with our new procedure, we have included detailed upgrade instructions in our guides. Our primary objective is to ensure a seamless experience for our developers. However, if you encounter any difficulties, please feel free to share them in our [troubleshooting discussion](https://github.com/solidusio/solidus/discussions/new?category=troubleshooting) or contact us on [Slack](https://solidusio.slack.com/):

For detailed upgrade instructions, please visit: https://guides.solidus.io/upgrading-solidus/v4.1

**Comment by zacksiri** [comment]
Congratulations! Solidus is the best thing for building e-Commerce apps!

**Reply by kennyadsl** [reply]
Thanks @zacksiri! ❤️
